tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -218,6 +239,15 @@ void ordering_test_cases() {
         static_assert((a1 <=> a1) == 0);
         static_assert((a2 <=> a0) < 0);
         static_assert((a0 <=> a2) > 0);
+
+        constexpr auto b1 = a1.begin();
+        constexpr auto b2 = a1.begin();
+        constexpr auto e1 = a1.end();
+
+        static_assert((b1 <=> b1) == 0);
+        static_assert((b1 <=> b2) == 0);
+        static_assert((b1 <=> e1) < 0);
+        static_assert((e1 <=> b1) > 0);

```

Comment:
These need to be non-`constexpr` variables and runtime `assert`s, per discussion on Discord. (Also 259-257.)

Reply:
Removed as we test the non constexpr case below